Sources/Plasma/FeatureLib/pfPatcher/pfPatcher.cpp

```diff
@@ -570,9 +572,15 @@ void pfPatcherWorker::IHashFile(pfPatcherQueuedFile& file)
     }
 
     // Check to see if ours matches
-    plFileInfo mine(file.fClientPath);
+    plFileName clientPathForComparison = file.fClientPath;
+    if (file.fFlags & kBundle) {
+        if (fFindBundleExe) {
```

Comment:
These could be combined into a single `if`:

```suggestion
    if ((file.fFlags & kBundle) && fFindBundleExe) {
```